### ISSUE TITLE
speed of BattleCruiser

### ISSUE THREAD

**Issue by EndrosG** [Amazinite/ES-Unfettered-Conquest#1]
Hey @Amazine, just found the BattleCruiser, which you obv. used in your arrow debris screenshot (I first though: Carrier with 3rd engine port) in your ES-Unfettered-Conquest collection and took a look at the definition files too.
Question: Is it purpose that the BC comes with only 130 engine capacity while having a 3rd engine port? From the graphics point, it should have a larger engine capacity than a carrier. I.e. I would have guessed at 230 or so. With only 130, it's max speed is far below any comparable ship. From the pure numbers, it is slower than
- a Behemoth freighter
- a Hauler I
- a Korath World Ship
- a Deep River is already "much faster"

Before writing this, I thought that you might have just looked at the acceleration and turning values of your fully equipped BC and compared it with a carrier having it's bays FULL.
I did not playtest it, but that's the only thing I could think of.
The speed index and maneuver index of your AC look great. It outruns many other ships in terms of max speed, while having a reasonable maneuverability. Looks good.

Compared to the default Navy Carrier, which has ten bays, I would really suggest to give it 230 engine cap, resulting in more choice when outfitting. (BC vs. vanilla C: -40 cargo space, +40 outfit space, +40 weapon capacity, -70 engine capacity ; +70 tons, +0.5 drag, +20 crew, +37 bunks).
To me it would even sound reasonable, if your BC did not just swap out -40 cargo space for +40 outfit space, but instead have more +outfit space lore-wise, as you remove 10 bays which represent quite some "cargo space" (as they are internal).
I have an excel sheet, where I calculate fighter bays with 80 tons and drone bays with 50 tons. But currently it's not up to date. But honestly I think that the "closing" of bays should really be compensated by some additional space.

**Comment by Amazinite** [comment]
Yeah, nothing here is final as of yet. I'm waiting for the Wanderer campaign to be finished and for development on the Republic campaign to pick up, then I'll start working on this more (and actually writing the story). The Battlecruiser originally had different stats than it does now, but I did a rework of a lot of the stats and never changed the images to reflect some of the changes like the Battlecruiser's small engine space. 

The Battlecruiser is meant to be a slow moving slugger able to go toe-to-toe with an Unfettered Solifuge (not counting fighters), so and increase in engine space would likely bring with it an increase in drag to compensate.

**Comment by EndrosG** [comment]
Is your BC supposed to be roughly equally sized with a carrier? So far I assumed so.
Looking at the "new" `Class C Freighter`, the 600 cargo space have dropped to 20 cargo space, some hardpoints have been removed, but 12 drone bays have been added.
That roughly fits to the 50 tons per drone bay which I have in my excel sheet from early 2016, when the Class C Freighter didnt exist yet.
Thus I would assume that your "modified carrier" should really gain some big space from removing 4 fighter bays and 6 drone bays. Approximately the same ~600 tons. Minus what it takes to create 37 additional bunks (36 = 9 x 4 = 9 x 20 tons = 180 tons).

I currently have to add those "new" human ships to my excel sheet, as well as the hai ships and the coalition ships.
And I intend to re-balance my modular ship design stuff from 2016 by making some sort of difference between interior and outer cargo space (and outfit space) modules. For weapon modules, engines modules and crew modules I have no need for change, since they are all placed directly behind the hull. - Reason is that I still think that my (own) civilian ships have too high shielding and hull values.

If you are interested, I might upload the current state of my marsian plugin to GitHub. Please let me know.
